### ISSUE TITLE
pytorch alternative install

### ISSUE THREAD

**Issue by jelka71** [ml-struct-bio/cryodrgn#271]
Dear Ellen, 

I have been struggling with installing cryoDRGN on both Ubuntu 22.04 and Rocky Linux 8, both ended up with similar problems where only the cpu version of pytorch was installed. 

The v1.8.0 trick does not work here as this is on newer GPUs where the compute capability is not supported by pytorch v1.8.0 
I tried almost all cuda version of pytorch in the pytorch repo, but most ended up with glibc incompatibility issues. 

What worked on the other hand was to install pytorch-gpu from the conda-forge repo. 

`conda install pytorch-gpu -c conda-forge`

Now I have a fully functional cryoDRGN install. 

It seems as if many people have pytorch issues with cryoDRGN, perhaps the version in the more generic conda-forge repo is a better choice for cryoDRGN?

Best, 
Jesper

**Comment by ic1062** [comment]
Thank you Jesper. This solved the problem for me on a CentOS 7.9/CUDA 11.4 workstation. Here are the details of my installation:

conda create --name cdrgn2.3 python=3.9 
conda activate cdrgn2.3 
conda install pytorch-gpu cudatoolkit=11.4 -c conda-forge 
git clone https://github.com/zhonge/cryodrgn.git 
cd cryodrgn 
pip install . 
cryodrgn --version 
   cryoDRGN 2.3.1.dev77+g3785153 


(INFO) (train_vae.py) (06-Jul-23 09:55:13) 59779090 parameters in model
(INFO) (train_vae.py) (06-Jul-23 09:55:13) 55832592 parameters in encoder
(INFO) (train_vae.py) (06-Jul-23 09:55:13) 3946498 parameters in decoder
(INFO) (train_vae.py) (06-Jul-23 09:55:13) Using 2 GPUs!
(INFO) (train_vae.py) (06-Jul-23 09:55:13) Increasing batch size to 16
(INFO) (train_vae.py) (06-Jul-23 09:55:43) # [Train Epoch: 1/74] [2000/148780 images] gen loss=0.558570, kld=8.931324, beta=0.125000, loss=0.558592
